source/usage-examples/replaceOne.txt

```diff
@@ -0,0 +1,90 @@
+.. _csharp-replace-one:
+
+====================
+Replace One Document
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can replace one document with another by using the ``ReplaceOne()`` synchronous method 
+or the ``ReplaceOneAsync()`` asynchronous method on a collection object.
+
+Example
+-------
+
+This example uses the following ``Restaurant`` class as a model:
+
+.. literalinclude:: ../includes/code-examples/replace-one/Restaurant.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language: csharp
+   :copyable:
+   :dedent:
+
+The following code replaces the first document in the ``restaurants`` collection that has a 
```

Comment:
Should we mention (here or somewhere else) that the replacement document will need to have the same ID as the old document? Or is that best left to the Fundamentals page? It _is_ seen in the code examples on this page.